sdk/python/feast/infra/offline_stores/snowflake.py

```diff
@@ -91,6 +91,9 @@ class SnowflakeOfflineStoreConfig(FeastConfigBaseModel):
     schema_: Optional[str] = Field(None, alias="schema")
     """ Snowflake schema name """
 
+    authenticator: Optional[str] = ""
```

Comment:
Got it on excessive authentication.

See following error in integration test:

"E       snowflake.connector.errors.DatabaseError: 390139 (08001): None: Failed to connect to DB: ***.snowflakecomputing.com:443, The specified authenticator is not accepted by your Snowflake account configuration.  Please contact your local system administrator to get the correct URL to use."

My guess is your code is always passing the authenticator kwarg, when it should only be passed if specified

authenticator: Optional[str] = ""
vs
authenticator: Optional[str] = None

would be my first guess